src/System.Text.Encodings.Web/src/Properties/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+// This attribute only exists to prevent the build system from tree shaking
```

Comment:
The proper way to do this is to `ILLinkTrim.xml` file. Please do not use InternalsVisibleTo

Reply:
Opened https://github.com/dotnet/corefx/pull/38356